src/com/massivecraft/factions/integration/V19/IntegrationV19.java

```diff
@@ -0,0 +1,31 @@
+package com.massivecraft.factions.integration.V19;
+
+import com.massivecraft.massivecore.Engine;
+import com.massivecraft.massivecore.Integration;
+
+public class IntegrationV19 extends Integration
+{
+    // -------------------------------------------- //
+    // INSTANCE & CONSTRUCT
+    // -------------------------------------------- //
+
+    private static IntegrationV19 i = new IntegrationV19();
+    public static IntegrationV19 get() { return i; }
+    private IntegrationV19()
+    {
+        this.setClassNames(
+            "org.bukkit.event.entity.AreaEffectCloudApplyEvent"
+            // TODO: Add other classes that only exist in Minecraft 1.9
+        );
+    }
+
+    // -------------------------------------------- //
+    // OVERRIDE
+    // -------------------------------------------- //
+
+    @Override
+    public Engine getEngine()
+    {
+        return EngineV19.get();
+    }
```

Comment:
Totally forgot,  we usually insert one blank line here between the last method {} and the class ending bracket }. So please do that ^^.